app/grants/templates/grants/components/card.html

```diff
@@ -98,7 +98,7 @@ <h2 class="grant-item__title font-subheader">
                   </span>
                 </div>
               </div>
-              <template v-if="cred.is_authenticated">
+              <template v-if="isUserLogged">
```

Comment:
@thelostone-mc this will not work this componet doesn't have `isUserLogged` (this only exist on grant details component) the prop for this component is named `cred`

Reply:
There wasn't a prop called cred causing this check to fail! 
I did see credentials but I wasn't able to access that as well